Dlr.sln

```diff
@@ -53,10 +52,6 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{3272E46C-F4D9-418D-9EE9-081A1B658EE4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Microsoft.Scripting.AspNet. I don't mind re-adding it, but it should be updated to the new csproj format if we do.